include/singa/core/device.h

```diff
@@ -49,123 +49,123 @@ namespace singa {
 /// Allocate memory and execute Tensor operations.
 /// There are three types of devices distinguished by their programming
```

Comment:
I think your editor changed this file automatically.
pls revert it back.

Reply:
yes